packages/server/scripts/generate-secrets.mjs

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env node
+
+import crypto from "node:crypto";
+import bcrypt from "bcryptjs";
+import { intro, password, note, outro, isCancel, cancel } from "@clack/prompts";
+
+function generateJWTSecret() {
+  return crypto.randomBytes(64).toString('hex');
+}
+
+async function generatePasswordHash(password) {
+  return await bcrypt.hash(password, 12);
```

Comment:
You should salt the password with the `jwtSecret`. Take a look at `bcrypt.hash` – it optionally takes a 2nd parameter as a salt.

Otherwise somebody could determine the original password by testing the hashed value against a rainbow table (rainbow table attack).

(After that change, `jwtSecret` won't be strictly used for generating JWTs anymore, so I would rename it something else like `CF_CRYPTO_SECRET` or something else.)

Reply:
Also reminder again to avoid magic numbers like `12`.